src/policy/policy.cpp

```diff
@@ -21,7 +21,7 @@ CAmount GetDustThreshold(const CTxOut& txout, const CFeeRate& dustRelayFeeIn)
     // need a CTxIn of at least 148 bytes to spend:
     // so dust is a spendable txout less than
     // 182*dustRelayFee/1000 (in satoshis).
-    // 546 satoshis at the default rate of 3000 sat/kB.
+    // 546 satoshis at the default rate of 3000 sat/kvB.
```

Comment:
in our case we don't have segregated witness txes; size of tx in bytes are equal to virtual size.

I think this backport 21752 is DNM

Reply:
reverted this change